Feature/rdkb log adaptations

morantr opened the pull request:
Log files adaptation for rdkb.

arnout commented on CMakeLists.txt:
```diff
@@ -135,6 +136,8 @@ elseif (TARGET_PLATFORM STREQUAL "rdkb")
     set(BEEROCKS_LOG_FILES_ENABLED "false")
     set(BEEROCKS_LOG_SYSLOG_ENABLED "true")
+    set(BEEROCKS_LOG_FILES_PATH "/rdklogs/logs")
+    set(BEEROCKS_LOG_FILES_SUFFIX ".txt.0")
```
Please explain the commit message why this is needed.

arnout commented on common/beerocks/scripts/CMakeLists.txt:
```diff
@@ -5,11 +5,16 @@ configure_file(
+        @ONLY
+        )
+
 set(scripts ${CMAKE_CURRENT_BINARY_DIR}/prplmesh_utils.sh)
+
 if (NOT TARGET_PLATFORM STREQUAL "linux")
-list(APPEND scripts beerocks_utils.sh
-    beerocks_endurance_test.sh beerocks_watchdog.sh
-    udhcpc.script.alt)
+list(APPEND scripts ${CMAKE_CURRENT_BINARY_DIR}/beerocks_watchdog.sh)
 endif()
```
Please also remove the unused scripts from the repository. It would be better to have a separate first commit that does that removal (both from CMakeLists.txt and from the repository), and a second commit that updates beerocks_watchdog.sh.

arnout reviewed (approved):
Nitpick comments only.

Subject of the first commit is a bit short. Maybe "cmake: add configuration for log files".

If you don't have time to fix them, you can just merge.

morantr committed:
cmake: Add configuration

Add log files suffix configuration, and override it with ".txt.0" on rdkb.
This is needed since the logs have a different suffix on RDKB.
Also, override logs file path on rdkb.

Signed-off-by: Moran Shoeg <moranx.shoeg@intel.com>

morantr committed:
common: Update prplmesh_utils.sh to take cmake configuration

Update the use of log files path and suffix to cmake configuration

Signed-off-by: Moran Shoeg <moranx.shoeg@intel.com>

morantr committed:
common: Fix paths on beerocks_watchdog

Fix beerocks_watchdog path according to cmake configuration.
This has required to chang the script to ".in" type, and modify
the cmake file to generate from it the script.
On the way, removed irrelevant scripts from Linux target and from the repo.

Signed-off-by: Moran Shoeg <moranx.shoeg@intel.com>

morantr committed:
cli: Fix logging configuration

Cli logging configuration is not configured like other beerocks processes.
This has caused to a configuration error on rdkb.
Aligned logging configuration to the way it is done on other processes.

Signed-off-by: Moran Shoeg <moranx.shoeg@intel.com>

morantr committed:
prplmesh_utils.sh: Add roll_logs command

Currently, the roll_lof functionality is broken since it uses old beerocks_utils.sh script that is not being used anymore.

Copy roll_log function from prplmesh_utils.sh

Signed-off-by: Moran Shoeg <moranx.shoeg@intel.com>

morantr committed:
controller/agent: Add prints on rdkb rdkb_extensions

Signed-off-by: Moran Shoeg <moranx.shoeg@intel.com>

morantr committed:
prpmesh_utils.sh: Fix prplmesh status regex expression

Apparently on some Linux version space is added to the process name.
Therefore added "($|[[:blank:]])" to the end of the regex expression which means
the end of the line ($) or blank character ([[:blank:]])

Signed-off-by: Moran Shoeg <moranx.shoeg@intel.com>
